### ISSUE TITLE
Notary Application: ZhongYiGuoLian 

### ISSUE THREAD

**Issue by Patapon0702** [filecoin-project/notary-governance#638]
# Notary Application

To apply to be a Fil+ Notary, please review the Notary Overview [here](https://github.com/filecoin-project/notary-governance/tree/main/notaries#overview) and then fill out the following form. 

## Core Information
- Name: Patapon
- Affiliated organization: ZhongYiGuoLian
- On-chain address to be notarized (recommend using a new address):f1fdyvsszg7cdltpq3xlasjmajdpmwhpolwqkqryy
- Country of Operation: China
- Region of Operation: Europe
- Use case(s) to be supported: Professional Services, Web 3.0 / Decentralized Applications, Web 2.0 Applications, Scientific Data, Media & Entertainment
- DataCap requested for allocation (10TiB - 10PiB):10 PiB would be preferred, as it would give us more flexibility with projects and supporting partners.
- Are you applying on behalf of yourself or an organization?: Organization

_Please respond to the questions below in paragraph form, replacing the text saying "Please answer here". Include as much detail as you can in your answer!_

### Time Commitment
Describe the nature and duration of your affiliation with the Filecoin network. Please include relevant GitHub handles, Storage Provider IDs, significant projects or contributions (with links).
```
As an organization, we built quite an extensive relationship with the Filecoin community since 2019. We have sustained some contributions to the overall network. 
As a miner and researcher，we joined the testnet, space race, slingshot, mainnet. 
Our miner ids are f01901815  f01890287  f01736786

```

### Stake Exposure
Please cite total token at stake (currently available, locked as collateral, vesting over time) and any substantiating evidence (i.e., addresses on chain with their corresponding FIL amounts).
```
I prefer not to disclose.
```

How did you acquire the FIL cited above?
```
I prefer not to disclose.
```

### In-protocol Reputation
Please describe (in detail) your activity and tenure as a member of the Filecoin community. Please note (with links where possible) any contributions made to implementations of Filecoin, the spec, documentation, or to substantially help the Filecoin ecosystem grow. 
```
In 2017, a core technical team from distributed storage professional researchers, blockchain experts, senior IT engineers and hardware engineers was established, dedicated to distributed storage promotion, 
distributed storage device research and development, distributed storage application development. At present, we have deployed multiple distributed storage data centers around the world, 
with multiple 10-gigabit backbone networks. At the same time, we are providing decentralized cloud computing services based on the Web 3.0 infrastructure backboned by blockchain technology. 
In the future, distributed storage and data centers will be deployed. 
```

### In-protocol Security
Please describe your contributions to the security of Filecoin and the duration over which you've made contributions. Please also include any links or references that can substantiate your contributions. 
```
None
```

### Organizational Reputation
Please describe the nature of your organization, including the country of registration, size of the organization, and time since inception.
```
Organization: Zhong Yi Guo Lian
Nature:  private company
Registration Country: Jiangsu, CN
Established: Dec, 2019
Size: over 100 employees
```

Please share any relevant details to help substantiate information about your organization (website, named officers, links to social media profiles).
```
website: https://zettachain.cn/about.html
Wechat official Account: zhongyiguolian
```

Please share any relevant external information regarding your organization (e.g. news articles, social media profiles, etc.)
```
https://m.sohu.com/a/465497344_537750
https://haokan.baidu.com/v?pd=wisenatural&vid=4153170789718018075
```

### Individual Reputation
Please share links to at least 2 of your (personal) social media profiles (or accounts that you are able to use) and the approximate size of your audience (i.e., followers, subscribers) for each one.
```
discord:APM#5342
Slack: patapon0702
```

# Allocation Plan [refer to the Rubric to see how this impacts your Notary score](https://docs.google.com/spreadsheets/d/172-sbd5qzdbSofvL_C5FHRyXEsTUpMiKspItygAVJA4/edit?usp=sharings)
### Allocation Strategy
How do you plan on allocating the DataCap requested above? Please describe your allocation strategy with as much specificity as you can. This includes the target amount per client and rate at which you'll allocate DataCap.
```
https://docs.google.com/spreadsheets/d/1oM_Urz8TwwiijQt8EZDxaNYe2yJR56PDXQYnYQvCnPo/edit#gid=0
The purpose of allocating the DataCap is to make sure clients use these DataCap in the right way to store real and valid data. We firstly specify some rules to figure out the true identity of the applicant.
• Browse their official website to get a general understanding of the company or the individual. 
• Query the company's registration information from government related websites to make sure the information he submitted is truth.
• Ask some questions to know about the applicant's position/organization/plan/use case etc. 
• Then We will classify applicants based on their industry reputation and scale etc. All applicants will be divided into eight categories to determine the amount of datacap they will get. 

The whole application process should be open and transparent. There are only several necessary internal processes Currently:
• Some brief communications via e-mail to make sure the e-mail address is authentic and valid.
• For some companies involved in confidentiality that can’t answer some details in public, we will have some communications in our private repository. In the event of a dispute, we will share the repository with officials immediately. The target and amount of our allocation can be found in the scoring guidelines. 

In addition, we will try to simplify the process to enable applicants interested in DataCap to use it asap. For customers who apply for a large amount of DataCap, we will review them more carefully.  
```

How do you plan on securing the DataCap to ensure your organization (and its delegated members) are the ones allocating the DataCap?
```
A hardware wallet will be used to store the keys associated with the DataCap allocation address securely. 

```

### Client Due Diligence
How will you vet the clients that are applying for DataCap? What questions will you ask to ensure your trust is placed well and that clients can properly handle the DataCap you intend to allocate to them? 
```
First of all, we will review the applicants, who need to upload relevant information, including the years of registration, industry influence and so on. After the approval, we will conduct regular verification. Customers who meet the requirements can continue to use or apply for a higher volume. Those who do not meet the requirements and do not provide reasons will be rejected. We will try to find out high-quality clients with long-term storage needs.

We have the corresponding audit, such as the allocation of policy table information, and communicate with customers of different levels on different issues, including (no limit) as follows:
① Team/personal introduction
② How much DataCap do you want, and why?
③ How will you allocate your DataCap?
④Have you ever allocated before? If yes, could you give us more details?
⑤ What will you do to avoid the datacap waste?
```

What processes will you employ when granting additional DataCap to a client that has previously been verified? This includes confirming that the client is not improperly using the DataCap they were previously granted, i.e., making deals with a single SP entity.
```
Periodically review and audit DataCap usage (as metioned above, records are tracked and reviewed periodically to determine disqualification or demotion of nonconforming customers)
Users can upgrade according to the actual situation to meet the requirements
Communicate with customers regularly to understand relevant requirements and progress, and provide better service to customers
Of course, try to avoid this situation in the initial review of the application and improve the initial application information supplement during the continuous record review process. 
```

### Bookkeeping Plan
Do you plan on conducting all your allocation decisions in public (e.g. Github repo), private (e.g. over email, Telegram, etc), or both? 
```
Yes, as mentioned above, all records will be public. Also, if we have detailed communication needs E-mail, we will save the screenshots.. 
```

Where do you plan on keeping a publicly accessible record of all your allocation decisions?
```
We would reserve our allocation record and make it public through our telegram：https://t.me/ifpsguild
```

### Engagement in Program
Which level (1-5) of service commitment are you willing to dedicate to participating in the Fil+ program? This includes making DataCap allocations (direct and/or Large Datasets), joining working groups, adding comments on discussion/issues, attendance in governance calls, messages in Slack, etc. 
```
I will spend two or three hours everyday to participate in this program. I will review every application carefully and make comments if I have question about the application. Then, I will allocate DataCap in time.

```

### Past allocation 
Have you previously received DataCap to allocate before? If so, please link to any previous applications.
```
None
```

## Disclosures
Do you/your organization have any relationship(s) with other existing notaries or their organizations? If yes, please list the names of the Notary individuals or organizations you may be related to.
```
None
```

**Comment by Kevin-FF-USA** [comment]
Hi @Patapon0702,

Confirming receipt. I'll be following up during the scoring rounds for additional questions and clarifications. Thank you

**Comment by Patapon0702** [comment]
Hi @Kevin-FF-USA 
Thank you for your feedback.

**Comment by Kevin-FF-USA** [comment]
Hi @Patapon0702, 

Thanks for applying to become a Notary in the Fil+ program! 

As we score your application, you will need to verify your contact information and affiliated organization.  Since you are applying on behalf of an organization, please email [NOTARYELECTION@FIL.ORG](mailto:NOTARYELECTION@FIL.ORG) from your Organizations email account. 
***This cannot be from your personal account. You Must include the address of your Organization.**  

Subject Line:  Notary Name, Application Number
Email Body:  
1. Your First and Last name
2. Your Organization
3. Your Country of Operation
4. Your Slack handle in [[filecoinproject.slack.com](http://filecoinproject.slack.com/)](http://filecoinproject.slack.com/)
5. Link to your GitHub application

**This step needs to be completed by 28NOV.**  Emails received after the 28th run the risk of delaying scoring and may result in your application not being completed.  Once we receive verification of your email, we will then send you a token code to verify and link your SLACK account.

Should you have any questions, please post them here in the comments section of your application and tag @Kevin-FF-USA and @panges2

Thank you!

**Comment by raghavrmadya** [comment]
Thank you for your verification information. At this stage, the Trust & transparency Working Group has flagged your application for further audit. The governance team will wait for guidance from the WG before we can proceed further with your application.

**Comment by Kevin-FF-USA** [comment]
Hi @Patapon0702 ,
We never received an email. Wanted to send a friendly follow up that this step is needed in order to verify your connection to the organization. Failure to reply will result in a decrease in your score and possible non-selection as a Notary.  If you have already replied (and we are overlooking), please paste the time stamp from sent message in this thread.

**Comment by Patapon0702** [comment]
Hi @Kevin-FF-USA 
I've already sent the email on Dec 24th.
Here is the time stamp
![微信图片_20221203093531](https://user-images.githubusercontent.com/114091955/205416243-f4073c9a-2770-4eb8-8948-0f867da215af.png)

**Comment by galen-mcandrew** [comment]
Please see attached for initial notary scoring by governance team. Thank you for applying!
[638.pdf](https://github.com/filecoin-project/notary-governance/files/10159331/638.pdf)

**Comment by Kevin-FF-USA** [comment]
@Patapon0702 

Thank you for applying to serve as a Notary in support of the Filecoin Plus program. After review of your application and a diligent scoring of your past and potential future contributions to the program, you have been selected to onboard as a Notary in this 4th round of elections. 

In the coming days you will receive an onboarding training checklist. It will walk you through the steps to verify your ledger verification in [https://plus.fil.org/](https://plus.fil.org/) as well as outline the best practices and responsibilities for a Notary in the Program.

Next steps

1. **By 20 December** - Complete the CONTACT AND DISCLOSURES form
    1. [https://airtable.com/shrjx52lvefitwdq6](https://airtable.com/shrjx52lvefitwdq6)
2. **By 30 December** - Ensure that the [Governance meeting time is on your calendar]( [https://calendar.google.com/calendar/embed?src=c_k1gkfoom17g0j8c6bam6uf43j0%40group.calendar.google.com&ctz=America%2FLos_Angeles](https://calendar.google.com/calendar/embed?src=c_k1gkfoom17g0j8c6bam6uf43j0%40group.calendar.google.com&ctz=America%2FLos_Angeles) ) and make scheduling plans which allow you to commit to attending per your SLA level expectations. ([L1-L5]( [https://github.com/filecoin-project/notary-governance/issues/630](https://github.com/filecoin-project/notary-governance/issues/630))) 

Please reach out to Galen, Kevin-FF, or Philop on Slack if you have any questions.

**Comment by Patapon0702** [comment]
@Kevin-FF-USA 
Hi，I just updated my ledger wallet address. Thank you.

**Comment by Patapon0702** [comment]
@Kevin-FF-USA Hi，I just updated my ledger wallet address. Can you help me with it?

**Comment by Patapon0702** [comment]
## Notary Ledger Verified
#### Message sent to Filecoin Network
> message CID: bafy2bzacedou2dis7rw7svgzu3jluobx7kcs45ftgjpk352y3accoeisvzqpi
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacedou2dis7rw7svgzu3jluobx7kcs45ftgjpk352y3accoeisvzqpi

**Comment by liyunzhi-666** [comment]
I also noticed this problem. The email address ending with @163.com is not an enterprise email address. I suggest to review this application. @Kevin-FF-USA @panges2 
cc @raghavrmadya

**Comment by Patapon0702** [comment]
@liyunzhi-666 @Carohere 
Hey Bro，I've already sent proofs to Philippe Pangestu, Kevin and RG in our private channel. Thank you for your concern.

**Comment by Carohere** [comment]
Ack. Haven't really gotten any of your responses these few months, hope you can understand my concerns. Would you mind sharing the screenshot sent by your business email tho?

**Comment by Patapon0702** [comment]
@Carohere 
Thank you for your work. Kevin and his team have already done these works. I've explained about what you concern to Kevin. If they still have problems, they will suspend my election.

**Comment by Carohere** [comment]
Hi @panges2, i received your DM months ago regarding this applicant but haven't heard anything after. Could you reveal more details?
